Week1_Projects/src/CircleOfArea.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class CircleOfArea {
+    public static void main(String[] args)
+    {
+        Scanner radius = new Scanner(System.in);
+
+        double r = radius.nextDouble();
+        final  double n = 3.14159;
```

Comment:
The value of pi should be declared as a **constant** variable with a meaningful name. Instead of using final double n = 3.14159; , it's better to use final double **PI** = 3.14159; or final double PI = Math.PI;